test/e2e/framework/ginkgowrapper.go

```diff
@@ -526,6 +526,21 @@ func withLabel(label string) interface{} {
 	return newLabel(label)
 }
 
+// WithFlaky specifies that a certain test or group of tests are flaky
```

Comment:
```suggestion
// WithFlaky specifies that a certain test or group of tests are failing randomly.
```

First, missing dot at the end (nit).

Second, the documentation needs to explain what "flaky" means because someone who doesn't know that is exactly the person who needs the documentation. The second sentence is good for that, but let's also explain "flaky" by using some different words for it.

In general, paraphrasing is often a good way to make documentation more useful.

Reply:
/lgtm cancel

Reply:
It's not that important, but let's fix it anyway, otherwise we'll never will...

Reply:
Thanks for the suggestion. Updated with this change.